### DISCUSSION TITLE
Unable to install dependencies on Mint 21.

### DISCUSSION THREAD

**Discussion by scruffylookingnerdherder** [sn4k3/UVtools#570] [Q&A]
I am having an issue with getting UVtools to load on Mint. When I go through requirements section and putting the code into the terminal I see the following issues:
```
E: Unable to locate package libdc1394-22
E: Unable to locate package libopenexr24
```
When trying to launch UVtools I get the message 

> "Your system doesn't have the required dependencies in order to run.
> Those dependencies are required at libcvextern/OpenCV library.
> UVtools is built on top of the OpenCV and therefore cannot run.
> 
> Please install or build the dependencies in order to run the software.
> Check the manual page at 'Requirements' section for help"

Does anyone know what I need to do to get this to install all the dependencies correctly? My apologies, I am still new to Linux so this may be obvious to more experienced users.

**Comment by sn4k3** [comment]
Ok you need the more recent packages since you may run a recent linux version.

try:
```
apt install libdc1394-25 -y
apt install libopenexr25 -y
```

Report back if both install and UVtoosl run

**Reply by scruffylookingnerdherder** [reply]
Terminal says both of those are already installed. No change when I try to run it. Any other ideas?

**Reply by sn4k3** [reply]
did you run the command as indicated on installation?

`wget -qO - https://raw.githubusercontent.com/sn4k3/UVtools/master/Scripts/libdl-solver.sh | sudo bash`
